docs/workspaces/index.md

```diff
@@ -221,6 +221,16 @@ If a file name appears in multiple locations, the following precedence takes eff
 
 - Files in the `workspace/` directory are considered last.
 
+## Maintained workspace images
+
+PrairieLearn provides and maintains the following workspace images:
+
+- [An Ubuntu 24.04 desktop](https://github.com/PrairieLearn/PrairieLearn/tree/master/workspaces/desktop/)
+- [JupyterLab with Python 3.11](https://github.com/PrairieLearn/PrairieLearn/tree/master/workspaces/jupyterlab-python/)
+- [RStudio with R version 4.4](https://github.com/PrairieLearn/PrairieLearn/tree/master/workspaces/rstudio/)
+- [VS Code with Python 3.10](https://github.com/PrairieLearn/PrairieLearn/tree/master/workspaces/vscode-python/)
+- [Terminal emulator based on xterm.js](https://github.com/PrairieLearn/PrairieLearn/tree/master/workspaces/xtermjs/)
```

Comment:
Let's use the workspace image names here, e.g.:

- [`prairielearn/workspace-jupyterlab-python`](https://github.com/PrairieLearn/PrairieLearn/tree/master/workspaces/jupyterlab-python/): JupyterLab with Python 3.11

Reply:
Just to clarify @nwalters512, would you like me to replace `JupyterLab with Python 3.11` with `prairielearn/workspace-jupyterlab-python` or keep both the workspace image name and the editorialised name?